scoped_model_counter/README.md

```diff
@@ -0,0 +1,26 @@
+# ScopedModel Counter
+
+The starter Flutter application, but using ScopedModel to manage state.
+
+This app is a direct counterpart to the [simple counter application][] that you get when you create
+a new Flutter project. That one uses a `StatefulWidget` to manage application state.
+The version in this repository uses a simple app state management approach, `ScopedModel`.
+
+[simple counter application]: https://flutter.io/docs/development/ui/widgets-intro#changing-widgets-in-response-to-input
+
+It shows how you might deal with state that is modified from outside the app (for example,
+state synchronized over network) and which needs to be accessed and changed
+from different parts of your app.
+
+## Getting Started
+
+The only important part of the app is the `lib/main.dart` file. It has comments that will walk you
+through it.
+
+For more information on the `scoped_model` package (where `ScopedModel` comes from), please
+[read its documentation](https://pub.dartlang.org/packages/scoped_model).
```

Comment:
For more information on the `scoped_model` package (...), <del>please [read its documentation](https://pub.dartlang.org/packages/scoped_model)</del>.

-> 

For more information on the `scoped_model` package (...), <ins>see the [package documentation](https://pub.dartlang.org/packages/scoped_model).</ins>

Reply:
Done.